config.py

```diff
@@ -39,6 +39,8 @@
 plot = True
 plot = False
 
+# specify as tuple (tmin, tmax) or as None
+crop = None
```

Comment:
Is this something we want to expose to the users?

Reply:
I think it's fine, because it's not a private method and it can also be helpful for some data ... for example if you have EEG data and start recording early to prepare impedance values ... you'll always crop the first few minutes.

I am not saying that it's done often, but "exposing" the `crop` method should be fine

Reply:
Okay, but it's still not clear to me as the user if it crops the raw, epochs or evoked.